src/utils/netlify-utils.ts

```diff
@@ -0,0 +1,95 @@
+import axios from "axios"
+
+import { config } from "@config/config"
+
+import logger from "@root/logger/logger"
+
+const GITHUB_ORG_NAME = config.get("github.orgName")
+const NETLIFY_ACCESS_TOKEN = config.get("netlify.accessToken")
+
+type NetlifySiteDetails = {
+  id: string
+  password: string
+  build_settings: {
+    repo_path: string
+    repo_branch: string
+  }
+}
+
+type UpdateNetlifySiteDetails = {
+  password: string
+  build_settings: {
+    stop_builds: boolean
+  }
+}
+
+// Retrieves all netlify sites which build from the given repo's staging branch
+const getNetlifySiteDetails = async (repoName: string) => {
```

Comment:
could we define our return type expectedly here? 
just to clarify ya, why are we returning an array here? wouldnt we only have 1 staging site?

Reply:
sure, defined in https://github.com/isomerpages/isomercms-backend/pull/828/commits/b8ecf27e5f57094f67e0bceb3967748599f533bb

I've opted to return an array just in case we have multiple repos building from the same repo and staging branch - just to be safe!